can/interfaces/pcan/pcan.py

```diff
@@ -5,12 +5,54 @@
 import logging
 import time
 from datetime import datetime
+import platform
 
 from typing import Optional
-from can import CanError, Message, BusABC
-from can.bus import BusState
-from can.util import len2dlc, dlc2len
-from .basic import *
+
+from ...message import Message
+from ...bus import BusABC, BusState
+from ...util import len2dlc, dlc2len
+from ...exceptions import CanError, CanOperationError, CanInitializationError
+
+
+from .basic import (
```

Comment:
We could also just use a namespace e.g. `basic.PCAN_BITRATES`.

Reply:
Sure, also possible. I found it to be redundant since everything is prefixed with `PCAN_` anyways and it makes the class code more readable. The imports are simple to skim anyhow.